specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -97,6 +97,114 @@ export const attachmentsText = createDictionary({
     'uk-ua': 'Показати форму',
     'de-ch': 'Formular anzeigen',
   },
+  multipleFilesSelected: {
+    'en-us': 'Multiple files selected',
+  },
+  fileSize: {
+    'en-us': 'File Size',
+  },
+  deleted: {
+    'en-us': 'Deleted',
+  },
+  noFile: {
+    'en-us': 'No File',
+  },
+  incorrectFormatter: {
+    'en-us': 'Incorrectly Formatted',
+  },
+  alreadyUploaded: {
+    'en-us': 'Already Uploaded',
+  },
+  alreadyDeleted: {
+    'en-us': 'Already Deleted',
+  },
+  skipped: {
+    'en-us': 'Skipped',
+  },
+  cancelled: {
+    'en-us': 'Cancelled',
+  },
+  frontEndInterruption: {
+    'en-us': '{action:string} was in progress when interruption occured',
+  },
+
+  matchError: {
+    'en-us': 'Match Error',
+  },
+  errorReadingFile: {
+    'en-us': 'Error reading file',
+  },
+
+  unhandledFatalResourceError: {
+    'en-us': 'Unhandled fatal resource error:',
+  },
+  attachmentImportDatasetsCount: {
+    'en-us': 'Attachment Import Datasets ({count:number})',
+  },
+  timeStampCreated: {
+    'en-us': 'Timestamp Created',
+  },
+  timeStampModified: {
+    'en-us': 'Timestamp Modified',
+  },
+  newAttachmentDataset: {
+    'en-us': 'Attachment Import Dataset: New',
+  },
+  uploadInterrupted: {
+    'en-us': 'Upload Interrupted',
+  },
+  uploadInterruptedDescription: {
+    'en-us':
+      'The upload was in progress when a system error occurred. Some files may have been uploaded.',
+  },
+  rollbackInterrupted: {
+    'en-us': 'Rollback Interrupted',
+  },
+  rollbackInterruptedDescription: {
+    'en-us':
+      'The rollback was in progress when a system error occured. Some files may have been deleted',
+  },
+  deleteAttachmentDatasetWarning: {
+    'en-us': 'Deleting attachment dataset is not recoverable',
+  },
+  status: {
+    'en-us': 'Status',
+  },
+  attachmentID: {
```

Comment:
we use strict camel case. This means no consecutively capitalized characters (because then it's unambiguous, and you have only one way of writing a word - super consistent)
Thus, `attachmentId`